docs/middleware/actions.md

```diff
@@ -99,6 +99,16 @@ This action has these additional `context` properties:
 
 > The reply about to be sent
 
+## `'receivePresence'`
```

Comment:
I'm not 100% sure on the naming of this hook — "receive" does sound a bit weird, because it makes me think that it's just before a client is about to receive a message, but from the server's point of view it does make sense.

The other thing that's slightly odd about the naming is that all of our other hook names are named in the convention that the action is about to happen (eg `apply` means an op is about to be applied to the snapshot), whereas here the receive has already happened. Could potentially rename to `afterReceivePresence`?

Reply:
Oh although actually this naming is consistent with the existing [`receive`](https://github.com/share/sharedb/blob/07f8bd98e5cdae37e925d4c0c709a1c3cbbca5e0/lib/agent.js#L341) hook, so maybe it's fine after all.